migrations/201910231115243-update-data-info-add-data-hash.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.addColumn(
+            'data_info',
+            'otjson_size_in_bytes',
+            {
+                type: Sequelize.INTEGER,
+            },
+        );
+
+        return queryInterface.addColumn(
+            'data_info',
```

Comment:
Column with the same name twice occurred?

Reply:
`data_info` is the table name, while `otjson_size_in_bytes` and `data_hash` are column names